.github/workflows/integration.yml

```diff
@@ -20,4 +20,4 @@ jobs:
     - name: Install OVS
       run: sudo apt-get install openvswitch-switch
     - name: Run integration tests
-      run: sudo make test-integration
+      run: sudo -E GO="$(go env GOROOT)/bin/go" make test-integration
```

Comment:
a comment may be nice here for posterity :)

also do you really need to preserve the whole environment with `-E`, or would `sudo GO="$(go env GOROOT)/bin/go" make test-integration` be enough?

Reply:
Thanks, I have updated accordingly.